<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrixGraphClass</class>
 <widget class="QMainWindow" name="matrixGraphClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图搜索</string>
  </property>
  <property name="windowIcon">
   <iconset resource="matrixGraph.qrc">
    <normaloff>:/matrixGraph/image/icon.png</normaloff>:/matrixGraph/image/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>611</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="showArea">
     <item>
      <widget class="QGroupBox" name="graphBox">
       <property name="title">
        <string>图</string>
       </property>
       <widget class="QLabel" name="graphLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>280</width>
          <height>420</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>420</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dsiplayBox">
       <property name="title">
        <string>显示框</string>
       </property>
       <widget class="QLabel" name="displayLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>280</width>
          <height>420</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>420</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="backArea" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>211</width>
      <height>426</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="ansBox"/>
     </item>
     <item>
      <widget class="QWidget" name="inputArea" native="true">
       <layout class="QVBoxLayout" name="inputLayout">
        <item>
         <widget class="QPushButton" name="inputAllBtn">
          <property name="text">
           <string>INPUT_ALL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="inputEachBtn">
          <property name="text">
           <string>INPUT_EACH</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="functionArea" native="true">
       <layout class="QGridLayout" name="functionLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="bfsNonrebtn">
          <property name="text">
           <string>BFS_NONRE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="dfsNonreBtn">
          <property name="text">
           <string>DFS_NONRE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="listShowBtn">
          <property name="text">
           <string>LIST_show</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dfsReBtn">
          <property name="text">
           <string>DFS_RE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="matrixGraph.qrc"/>
 </resources>
 <connections/>
</ui>
